consistent-sampling/src/main/java/io/opentelemetry/contrib/sampler/consistent56/ConsistentParentBasedSampler.java

```diff
@@ -27,19 +35,40 @@ final class ConsistentParentBasedSampler extends ConsistentSampler {
    *
    * @param rootSampler the root sampler
    */
-  ConsistentParentBasedSampler(ConsistentSampler rootSampler) {
+  ConsistentParentBasedSampler(ComposableSampler rootSampler) {
```

Comment:
Likewise, the OTel ParentBased sampler should be updated to this behavior, I think.